client/src/components/Reviews/Reviews.jsx

```diff
@@ -1,139 +1,142 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+//Components
 import ReviewMapper from './ReviewMapper.jsx';
 import MetaReview from './MetaReview.jsx';
 import Form from './Form.jsx';
-
-import { Container } from '../styles/Container.styled.js';
+import SearchBar from './SearchBar.jsx';
+//styled components
 import { Button } from '../styles/Button.styled.js';
+import { Grid, Row, Col, Scroll } from '../styles/reviewstyles/reviewWidget.styled.js';
 
 
-
-//hooks only in stateless components aka not classes
-//reviews
-
 const Reviews = ({ currentProductId, currentProductRating, totalReviews }) => {
+
+  //hooks related to the review list and restrictions
   const [startPoint, setStart] = useState(0);
+  const [filterRating, setFilterRating] = useState([]);
+  const [noMoreResults, setnoMoreResults] = useState(false);
 
-  //reviewList states and hooks
+  //hooks related to controlling parameters for review GET requests
+  const [count, setCount] = useState(10);
   const [reviewList, setReviewList] = useState([]);
-  const [reviewPage, setPage] = useState(1);
-  const [catchUpPage, incPage] = useState(1);
-  const [moreResults, noMoreresults] = useState(true);
-  const [sort, toggleSort] = useState('newest');
+  const [dropDownSort, setDropDownSort] = useState('relevant');
 
-  //metaReview states and hooks
+  //metaReview and form hooks
   const [metaReview, setMetaReview] = useState({});
+  const [totalRatings, setTotalRatings] = useState(0);
   const [form, formClick] = useState(false);
 
-  let getTwoReviews = (id) => {
-    axios.get('/reviews', { params: { id: id, page: reviewPage, sort: sort } })
+  //search hook
+  const [currentSearch, setCurrentSearch] = useState('');
+
+  //Submits GET request to results and obtains 2 new reviews
+  let getTwoReviews = () => {
+    axios.get('/reviews', { params: { id: currentProductId, count: count, sort: dropDownSort } })
       .then(results => {
-        if (results) {
+        if (results.data.results) {
           let updatedPage = reviewList.concat(results.data.results.slice(startPoint, (startPoint + 2)));
           setReviewList(updatedPage);
           setStart(prevState => prevState += 2);
         } else {
-          noMoreresults(prevState => false);
+          setnoMoreResults(prevState => true);
```

Comment:
You don't need to use prevState. You can put setnoMoreResults(true);

Reply:
changed. Thank you!